modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -59,7 +59,7 @@ async fn all_advisories(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
     let (storage, _) = FileSystemBackend::for_test().await?;
 
     let app = actix_web::test::init_service(
-        App::new().configure(|config| configure(config, db, storage.clone())),
+        App::new().configure(|config| configure(config, db, storage.clone(), None)),
```

Comment:
What is the extra parameter buying us?

Reply:
That we can authenticate only a scope vs all of the server endpoints. Which would include the UI, which actually is there to trigger then authentication.